doc/dev/testing/travis-build.md

```diff
@@ -112,7 +112,6 @@ The markdown test does not create a new cluster and runs in a barebones travis V
 [pr-builds]: https://travis-ci.org/operator-framework/operator-sdk/pull_requests
 [script]: ../../../hack/ci/setup-openshift.sh
 [sanity]: ../../../hack/tests/sanity-check.sh
-[unit]: ../../../hack/tests/unit.sh
```

Comment:
If you removed the link to the file, you also need to remove the reference to this link. Change the readme https://github.com/operator-framework/operator-sdk/blob/b23e3e145904bdef0124830d8372d5d53a59b771/doc/dev/testing/travis-build.md#go-tests

Reply:
@LiliC updated the doc, thanks for catching that.